mrbgems/mruby-fiber/src/fiber.c

```diff
@@ -173,6 +174,22 @@ fiber_switch_context(mrb_state *mrb, struct mrb_context *c)
   mrb->c = c;
 }
 
+static mrb_value
+fiber_error_static(mrb_state *mrb, const char *mesg)
```

Comment:
I, as an insider, can imagine this `_static` means "calls mrb_str_new_static() inside", but I think it's too much to expect for others. Probably we should remove `_static` and add a comment that "`mesg` should be a literal string" or something similar.

Reply:
:black_cat: Surely, three months from now I will be doing something.
I will fix it.